redirects-list.json

```diff
@@ -1,5 +1,11 @@
 [
-  "^https://www\\.cml\\.dev(.*)?                                                          https://cml.dev$1",
-  "^/(?:docs|documentation)(/.*)?$                                                        /doc$1",
-  "^/(.+)/$                                                                               /$1"
+  "^https://(?:www\\.)?cml\\.dev(.*)?                                      https://cml.dev$1",
+  "^/(?:docs|documentation)(/.*)?$                                         /doc$1",
+
+  "^/doc/start/start-github(/.*)?$                                         /doc/start/github",
+  "^/doc/start/start-gitlab(/.*)?$                                         /doc/start/gitlab",
+
+  "^/doc/install(/.*)?$                                                    /doc/install-with-npm 303",
```

Comment:
why 303 in this case specifically? we are going to change this soon?

Reply:
yes